### ISSUE TITLE
老师您好，我修改如下问题中的参数，将pre_len的参数修改后，实验结果并没发生变化，请问如何预测30,45.60分钟的交通流呢？

### ISSUE THREAD

**Issue by TigerJyf** [lehaifeng/T-GCN#63]
def __init__(
        self,
        model: nn.Module,
        regressor="linear",
        loss="mse",
        pre_len: int = 1,
        learning_rate: float = 1e-3,
        weight_decay: float = 1.5e-3,
        feat_max_val: float = 1.0,
        **kwargs
    ):
